apps/vc-api/Dockerfile

```diff
@@ -28,9 +28,15 @@ RUN cp -R ./apps/vc-api/dist ./common/deploy/apps/vc-api
 
 FROM base as final
 ENV NODE_ENV=production
+
 EXPOSE 3000
 
 COPY --from=builder /app/common/deploy /app
 COPY license-header.txt /app
 
+ENV DB_TYPE=SQLITE_INMEMORY
```

Comment:
This variable is also set in `.env`. Can we skip it here?

Reply:
The idea is to have this change backward compatible with previous deployments of the SSI. Now, they use an in-memory SQLite database. This line ensures that if a container is started without DB_TYPE set, it executes as before.